website/blog/2025-09-29-typing-numpy.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Bringing NumPy's type-completeness score to nearly 90%
+description: We tell the story of how we brought NumPy's type-completeness score from ~33% to nearly 90%
+slug: numpy-type-completeness
+authors: [marcogorelli]
+tags: [typechecking, news]
+hide_table_of_contents: false
+---
+
+Because [NumPy](https://numpy.org/) is one of the most downloaded packages in the Python ecosystem, any incremental improvement can have a large impact on the data science ecosystem. In particular, improvements related to static typing can improve developer experience and help downstream libraries write safer code. We'll tell the story about how we helped bring its type-completeness score to nearly 90% from an initial 33%.
```

Comment:
Do we need to be a bit clearer about who "we" is referring to here? As this is going on the pyrefly website I'm wondering if some folks might assume this is purely the pyrefly team that has been doing this work. Maybe somewhere in the intro it should be more clear that "we" means "Quansight with support from Meta's pyrefly team" or similar. Curious what others think though. maybe @lolpack cares to weigh in?